packages/x-date-pickers/README.md

```diff
@@ -26,6 +26,45 @@ This component has the following peer dependencies that you will need to install
 },
 ```
 
+You need to provide a date-library that is used by the pickers by setting the `dateAdapter` to an adapter of your choosing.
+
+We currently support 4 different date-libraries:
```

Comment:
I think this statement is misleading (even if we already have it in the doc)

We re-export 4 adapters, and I guess we can say that we officially support only those 4 and that the others can be considered as less safe.

But we have users on Moment Hijiri, Moment Jalaali, Date Fns Jalali and JS Joda.

Reply:
I'm even wondering why are we re-exporting those Adapters? It would be easier to say that we use @date-io adapters and let developers import the adapter from date-io

Reply:
Probably so that people only have one package to install.
It was already the case in [material-ui-pickers](https://github.com/mui/material-ui-pickers/blob/next/lib/adapter/dayjs.ts)

I guess that's something we could revise in the future

Reply:
I added a sentence to notify the other libraries supported by `@date-io`

I noticed that adapters are in the dependencies. So I assume installing `@mui/x-date-picker` also install the 4 `@date-io/xxx` package.

The next instruction should not be `npm install @date-io/date-fns` but `npm install date-fns`